crates/air-lib/test-utils/src/native_test_runner.rs

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use super::CallServiceClosure;
+use air_interpreter_interface::RunParameters;
+use avm_server::avm_runner::*;
+
+// Borrowed from private module in the avm/server/src/interface/call_service_result.rs
```

Comment:
Let's make it public?

Reply:
Well, I wasn't aware it is re-exported, so, making it public is enough.